src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -494,6 +501,13 @@ std::string ContextAttribute::render
         valueIsNumberOrBool = true;
         break;
 
+#if 0
```

Comment:
@kzangeli , this is something that I feel you could need in you next PR. In negative case, remove this block upon next PR completion.

NTC